<launch>
<remap from="/scan" to="/scan_input"/>
<remap from="/scan_fixed" to="/scan"/>
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
  <param name="serial_port"         type="string" value="/dev/ttyUSB1"/>
  <param name="serial_baudrate"     type="int"    value="115200"/>
  <param name="frame_id"            type="string" value="laser_frame"/>
  <param name="inverted"            type="bool"   value="true"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  </node>
  <node name="ignore_laser" pkg="nav_main" type="ignore_laser" output="screen" >
       <param name="ignore_array" type="string" value="-176 ,-166, -129,-119, -85, -67, -56, -46, -10,5" />
     </node>
<node pkg="tf" type="static_transform_publisher" name="base_link_to_laser4"
    args="0.1 0.0 0.2 -1.57 0.0  0.0 /base_footprint /laser_frame 40" />
</launch>
